src/sentry/integrations/vercel/uninstall.py

```diff
@@ -60,35 +93,33 @@ def delete(self, request):
             integration.metadata["webhook_id"] = next_config["webhook_id"]
             integration.metadata["installation_id"] = next_config_id
 
+        integration.save()
+
+        # At this point we can consider if len(orgs) > 1. We have already updated the
```

Comment:
We did a query to see how many Vercel integrations have multiple Sentry organizations tied to it and the result was very low (`< 30`). So this is not a common case and given Vercel's changes, no long a possible one for new installations.

Reply:
For now until someone requests this feature from Vercel and they have to re-add it lol.